src/ExtensionSet.php

```diff
@@ -27,28 +27,37 @@
  */
 final class ExtensionSet
 {
-    private $extensions;
-    private $initialized = false;
-    private $runtimeInitialized = false;
+    /**
+     * @var ExtensionInterface[]
+     */
+    private array $extensions;
+    private bool $initialized = false;
+    private bool $runtimeInitialized = false;
     private $staging;
-    private $parsers;
-    private $visitors;
+    /**
+     * @var TokenParserInterface[]
+     */
+    private array $parsers;
+    /**
+     * @var NodeVisitorInterface[]
+     */
```

Comment:
```suggestion
     /** @var list<NodeVisitorInterface> */
```

Reply:
I've moved the other ones to multiplines to be consistent with all the other files.